scripts/memory/local.py

```diff
@@ -28,10 +28,20 @@ class LocalCache(MemoryProviderSingleton):
     def __init__(self, cfg) -> None:
         self.filename = f"{cfg.memory_index}.json"
         if os.path.exists(self.filename):
-            with open(self.filename, 'rb') as f:
-                loaded = orjson.loads(f.read())
-                self.data = CacheContent(**loaded)
+            try:
```

Comment:
This one isn't related to the debug flag issue. Submit as a separate PR.

Reply:
Hi @nponeccop I moved this out to https://github.com/Torantulino/Auto-GPT/pull/913